src/workloads/workloads.csproj

```diff
@@ -125,17 +127,35 @@
       <Output TaskParameter="Msis" ItemName="Msis" />
     </CreateVisualStudioWorkload>
 
-    <!-- Build all the SWIX projects. This requires full framework MSBuild-->
-    <MSBuild Projects="@(SwixProjects)" Properties="SwixBuildTargets=$(SwixBuildTargets);ManifestOutputPath=$(VSTemp)%(SwixProjects.SdkFeatureBand)" />
+    <!-- Split SWIX projects for packs and components/manifests and build them into separate folders. This allows us to consume pack-only drops
+         across multiple VS builds to support multi-targeting. -->
+    <ItemGroup>
+      <SwixWorkloadPackProjects Include="@(SwixProjects)" Condition="'%(PackageType)' == 'msi-pack'"
+                                ManifestOutputPath="$(VStemp)\p\%(SwixProjects.SdkFeatureBand)"
```

Comment:
How come nested in a p directory?

Reply:
Is it to avoid path length issues?

Reply:
yup, p => packs, c => components... We've introduced additional naming elements for multitargeting and if we ever do SxS preview feature bands, then the package IDs can get quite long so I'm just avoiding things early on. And we already include the `packs` and `component` parts in the final artifact name, but we need separate folders to zip things up insolation.